pkg/cacheutil/memcached_client.go

```diff
@@ -463,3 +464,8 @@ func (c *memcachedClient) resolveAddrs() error {
 
 	return c.selector.SetServers(servers...)
 }
+
+func (c *memcachedClient) getServerAddrByKey(key string) net.Addr {
+	addr, _ := c.selector.PickServer(key)
```

Comment:
could we be strict check the error? Or at least document why we are 100% confident the error does not need checking?

Reply:
IMO if this fails we just log empty addr, which is fine, no?

Reply:
> IMO if this fails we just log empty addr, which is fine, no?

Correct. That was my intention. What's your take @squat ?

Reply:
that's OK but nothing is preventing anyone from using this internally and getting surprised later when something doesn't work and there's no error. Any reason this really needs to be a new func vs simply inlined above 291? Otherwise I would suggest to add this exact thing as a comment :)

Reply:
Right. I removed the function. What's about now?